agent/01_agent_requirements.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+source $SCRIPTDIR/common.sh
+
+if [[ -z ${E2E_TEST_SCENARIO} ]]; then
```

Comment:
For consistency with the other steps, it would be better to invoke the `early_deploy_validation` func at the beginning

Reply:
Done